README.md

```diff
@@ -6,6 +6,8 @@ Tool for verifying local and external references in repository documentation.
 
 For now, support for markdown files only is provided.
 
+<img src="https://imgur.com/2w8FVIH.png" alt="Output sample" width="600"/>
```

Comment:
Imo, it's more reliable to put the file with this picture into the repo, then it will work even if imgur dies.

Reply:
Hm, that picture weights 64k, and we may need to update it often which means that picture will take a considerable amount of space in the repository.
Maybe as compromise upload this picture to gist? Probably we can assume that GitHub working `<=>` gists working.

Reply:
Okay, let's do gist :man_shrugging: